.travis.yml

```diff
@@ -9,6 +9,7 @@ services:
 before_script:
   - yarn db:setup:user
   - yarn db:reset
+  - yarn db:seed
 script:
   - yarn test
```

Comment:
I would move `yarn lint` first before `yarn test` as most of the linting should happen before we test

Reply:
+1 to Henry's comment